docs/src/main/asciidoc/security-getting-started.adoc

```diff
@@ -3,70 +3,70 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= Getting Started with Quarkus Security
+= Secure a Quarkus application with basic authentication and JPA
 
 include::./attributes.adoc[]
 
-This guide demonstartes how you can quickly create a secure Quarkus application by using xref:security-built-in-authentication.adoc#basic-auth[Basic AuthenticationMechanism] and `JPA IdentityProvider` to create `SecurityIdentity` which is authorized by the `Role Based Access Control` (RBAC) layer before the access to application is permitted.
-
+To help you get started with security in Quarkus, this tutorial demonstrates how you can further secure your Quarkus application endpoints by combining the xref:security-built-in-authentication.adoc#basic-auth[Quarkus built-in basic HTTP authentication] with the JPA identity provider.
+`IdentityProvider` verifies the authentication credentials to create a `SecurityIdentity` instance and authorizes a secure connection to the endpoint by using Role Based Access Control (RBAC). 
```

Comment:
It reads good, but `and authorizes a secure connection to the endpoint` can confuse users - as a secure connection is often associated with `HTTPS` but not with a specific authentication mechanism. Besides we also have an `MTLS` authentication mechanism.
So IMHO we should say `and authorizes a secure access to the endpoint`

Reply:
+1. Removed the article 'a' as per grammar rules. But if this changes the accuracy we can rephrase as needed.